game/server/flights/routes.py

```diff
@@ -39,19 +40,23 @@ def commit_boundary(
     flight_id: UUID, game: Game = Depends(GameContext.require)
 ) -> LeafletPoly:
     flight = game.db.flights.get(flight_id)
-    if not isinstance(flight.flight_plan, PatrollingFlightPlan):
-        return []
-    start = flight.flight_plan.layout.patrol_start
-    end = flight.flight_plan.layout.patrol_end
-    if isinstance(flight.flight_plan, CasFlightPlan):
+    if isinstance(flight.flight_plan, CasFlightPlan) or isinstance(
+        flight.flight_plan, AirAssaultFlightPlan
```

Comment:
CTLD has a so called WaypointZone which is used so that dropped troops (as long as they are dropped inside this waypointzone) will head towards the center of this waypoint Zone. To make it visible to the liberation user how big this waypoint zone is is used the engagement radius for this purpose. Now you have a circle showing you where that waypoint zone is and you can then accordingly plan your drop off waypoint (where the helos will land). Otherwise it would be guessing. I already have written the docs about it here: https://github.com/dcs-liberation/dcs_liberation/wiki/Mission-planning#air-assault

In the screenshot you can see the ASSAULT waypoint (which will be used to create this waypointzone at that position) and the DROP OFF Waypoint :)
![grafik](https://user-images.githubusercontent.com/6678803/167815277-f41b72f3-44b0-4a01-afab-35eda0a23420.png)